src/System.Text.Json/src/System/Text/Json/Serialization/JsonCamelCaseNamePolicy.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Json.Serialization
+{
+    internal sealed class JsonCamelCaseNamePolicy : JsonPropertyNamingPolicy
+    {
+        protected override string ConvertName(string name)
```

Comment:
@jamesnk do your stuff.

Reply:
I don't have anything to discuss about this other than https://github.com/dotnet/corefx/pull/36940#discussion_r276018474 😬